screens/newLeague.js

```diff
@@ -63,14 +65,25 @@ export default class NewLeague extends Component {
     else {
       const startPos = parseInt(this.state.buyPower);
       createLeague(this.state.leagueName, startPos,
-        this.state.startDate, this.state.endDate).then((res) => {
-          joinLeague(res.data.inviteCode, this.state.nickname).then((res) => {
-            // TODO: Set current portfolio in store to joined portfolio
+        this.state.startDate, this.state.endDate).then(async (createRes) => {
+          joinLeague(createRes.data.inviteCode, this.state.nickname).then(async (joinRes) => {
+            if (joinRes.data.leagueId !== createRes.data.id) {
+              throw new Error({
```

Comment:
Does this tell the user any information? Does it cause the app to crash?

Reply:
Nah, this case shouldn't happen though it might give the user the wrong information, though it shouldn't happen since I'm using the leagueId to join the league. I'll just take it out because it's unnecessary.